build.sh

```diff
@@ -151,6 +155,10 @@ function usage()
     echo "                                 binding. Default 10MB                          "
     echo "    --osx-shmem-shmmax <size>    Maximum segment size for shared memory in OSX  "
     echo "                                 in bytes. Default 400MB                        "
+    echo "    --ns-resolution              Try to use the system real-time clock to get   "
+    echo "                                 nano-second resolution.                        "
+    echo "                                 For the Classic C++ Implementation only.       "
```

Comment:
You said "Classic C++ Implementation only" but I also see it enabled on the CPP moderm.

Reply:
Where? It should not be